### PR TITLE
Basic doc for setting up a dev environment (contribution to #183)

### PR DESCRIPTION
**Pull request by devium**
This only contains package installation instructions for Ubuntu/Debian. Other systems TODO.

#183

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,6 +43,51 @@ All PRs should be:
...
+#### Get the code
+
+    git clone https://github.com/raiden-network/raiden.git
+    cd raiden
+
+#### Setup
+
+    pip install -r requirements-dev.txt
+
+If you are installing these packages globally you might need to run this command with `sudo` privileges. Alternatively, you might want to consider creating a new `virtualenv`.
```
Comment (ulope) [inline CONTRIBUTING.md]: I would suggest to not even mention the option of installing globally and "force" people to use a virtualenv. 
There is just too much that can go wrong when people have "contaminated" system site-packages.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -43,6 +43,51 @@ All PRs should be:
...
+### Raiden
+
+#### Get the code
+
+    git clone https://github.com/raiden-network/raiden.git
+    cd raiden
+
+#### Setup
+
+    pip install -r requirements-dev.txt
```
Comment (ulope) [inline CONTRIBUTING.md]: Please add `pip install -e .` here, otherwise the `raiden` cli script won't get generated

File: CONTRIBUTING.md (hunk 1 above)
Comment (LefterisJP) [inline CONTRIBUTING.md]: +1 on not even mentioning global install.

**Comment by devium** [comment]
Fixed.

**Pull request merged**